clerk/organizations_test.go

```diff
@@ -199,6 +202,80 @@ func TestOrganizationsService_ListAll_invalidServer(t *testing.T) {
 	}
 }
 
+func TestOrganizationsService_UpdateLogo(t *testing.T) {
+	client, mux, _, teardown := setup("token")
+	defer teardown()
+
+	organizationID := "org_123"
+	uploaderUserID := "user_123"
+	expectedResponse := fmt.Sprintf(`{"id":"%s"}`, organizationID)
+	filename := "200x200-grayscale.jpg"
+	file, err := os.Open(fmt.Sprintf("../testdata/%s", filename))
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer file.Close()
+
+	mux.HandleFunc(
+		fmt.Sprintf("/organizations/%s/logo", organizationID),
+		func(w http.ResponseWriter, req *http.Request) {
+			testHttpMethod(t, req, http.MethodPut)
+			testHeader(t, req, "Authorization", "Bearer token")
+			// Assert that the request is sent as multipart/form-data
+			if !strings.Contains(req.Header["Content-Type"][0], "multipart/form-data") {
+				t.Errorf("expected content-type to be multipart/form-data, got %s", req.Header["Content-Type"])
+			}
+			defer req.Body.Close()
+
+			// Check that the file is sent correctly
+			fileParam, header, err := req.FormFile("file")
+			if err != nil {
+				t.Fatal(err)
+			}
+			if header.Filename != filename {
+				t.Errorf("expected %s, got %s", filename, header.Filename)
+			}
+			defer fileParam.Close()
+
+			var (
+				got  []byte
+				want []byte
+			)
+			_, err = fileParam.Read(got)
+			if err != nil {
+				t.Fatal(err)
+			}
+			_, err = file.Read(want)
```

Comment:
❓ Isn't the file handle already seeked to its end within the `UpdateLogo` function? I think you may need to seek to the start or re-open? 🤔

Reply:
Updated to move reset the seeker.